back/src/forms/resolvers/mutations/markAsResealed.ts

```diff
@@ -43,6 +43,9 @@ const markAsResealed: MutationResolvers["markAsResealed"] = async (
     ...updateInput
   });
 
+  // check companies type in TD
```

Comment:
No big deal mais je trouve ce genre de commentaire me semble un peu redondant. Je trouve qu'un commentaire aide vraiment quand il décrit l'intention d'une fonction au nom ambigu (ou que la logique est complexe et dure à suivre). Ici par exemple ça pourrait être "make sure the companies have the appropriate types" mais potentiellement le nom de la fonction est suffisant.

#my2cents

Reply:
Yes tu as raison le commentaire n'apportait rien ici, je l'ai supprimé